src/main/resources/db/migration/all/20241128163133__application_controller_and_linked_services_in_cas2bail.sql

```diff
@@ -0,0 +1,157 @@
+
+CREATE TABLE cas_2_bail_application_json_schemas
+(
+    json_schema_id UUID NOT NULL,
+    CONSTRAINT pk_cas_2_bail_application_json_schemas PRIMARY KEY (json_schema_id)
+);
+
+CREATE TABLE cas_2_bail_application_notes
+(
+    id                          UUID NOT NULL,
+    application_id              UUID,
+    created_at                  TIMESTAMP WITHOUT TIME ZONE,
+    body                        VARCHAR(255),
```

Comment:
we should use text

it's not recommended to use VARCHAR - https://wiki.postgresql.org/wiki/Don't_Do_This#Don.27t_use_varchar.28n.29_by_default

Reply:
Yes done.